codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/rds/AuthTokenGeneratorIntegration.kt

```diff
@@ -68,7 +68,7 @@ class AuthTokenGeneratorIntegration : SwiftIntegration {
                     username: String,
```

Comment:
Modifies codegen integration for RDS auth token generation to use modified function name in runtime auth token generator